Graphics/GraphicsEngineOpenGL/CMakeLists.txt

```diff
@@ -190,6 +201,7 @@ elseif(PLATFORM_MACOS)
     set(PRIVATE_DEPENDENCIES ${PRIVATE_DEPENDENCIES} glew-static ${OPENGL_LIBRARY})
 elseif(PLATFORM_IOS)
     set(PRIVATE_DEPENDENCIES ${PRIVATE_DEPENDENCIES} ${OPENGLES})
+elseif(PLATFORM_EMSCRIPTEN)
```

Comment:
I'm not the one who wrote this, but it seems odd to me that it does not
`set(PRIVATE_DEPENDENCIES ${PRIVATE_DEPENDENCIES} ${OPENGLES})` here?